### ISSUE TITLE
Suggestion: submit this program to alternativeto.net

### ISSUE THREAD

**Issue by zoidy** [manutalcual/winredock#4]
I spent a long time searching for a tool that would do what WinReDock does but all cost a lot of money and had a ton of features I didn't want. Adding this program to alternativeto.net as an alternative to programs like DisplayFusion or [UltraMon ](https://alternativeto.net/software/ultramon/) would increase it's visibility.

**Comment by manutalcual** [comment]
I would be glad adding it as an alternative, but I don't really think it is ready for mass consumption.
May be developing the improvements you proposed can make it closer to public, but another problem (for me) is that I haven't too much time to develop, test and adapt if there are many requests.
Let's improve, step by step, following your requests and we will see if it is ready for publishing.

Thanks!

**Comment by manutalcual** [comment]
I'm closer to mass consumption!
If I get positive feedback I'll consider to upload it to alternativeto.net

Thank you!

**Comment by manutalcual** [comment]
Can you test the last binary (loggin or not) to see if it works for you?
The branch is I0010-restoring-positions-doesnt-work.
If it works I may create an installer and upload somewhere to have it for alternativeto.net.

**Comment by zoidy** [comment]
Sure. I will test using the non-logging version throughout the week. If you create an installer, I'd suggest leave the option to download the just the binary as well. Any reason you don't use the Github "Releases" tab to distribute the binaries?

Thanks for your work!

**Comment by manutalcual** [comment]
Yep! I'm not using the release tab because it is in development state and I don't consider this as a distributable software yet.
If you give your OK I'll create the alpha release!
